lib/drivers/abbott/freeStyleLibreProtocol.js

```diff
@@ -610,13 +610,13 @@ export default class FreeStyleLibreProtocol {
         return errorCallback(new Error('Invalid database record checksum.'), data);
       }
 
-      dbRecords = dbRecords.split('\r\n');
+      dbRecords = dbRecords.split('\r\n').filter(Boolean);
 
       if (dbRecordCount !== dbRecords.length) {
         return errorCallback(new Error(`Invalid database record count: ${dbRecordCount} != ${dbRecords.length}`), data);
       }
 
-      return successCallback(null, dbRecords);
+      return successCallback(dbRecords);
```

Comment:
This signature is different. Does this fix a bug? Was data previously not being returned?

Reply:
`getDbRecords` is not actually being used in the Libre driver, as the Libre uses the binary protocol and not the text protocol. This is documented in the comments on lines 623 and 630, but not in the function itself. To be sure, I'm going to add a smoke test of the Libre reader to the test strategy.